baremetal_network_provisioning/tests/unit/ml2/extensions/test_bnp_switch.py

```diff
@@ -34,6 +34,7 @@
 
 TARGET_PLUGIN = 'neutron.plugins.ml2.plugin.Ml2Plugin'
 _get_path = test_base._get_path
+extensions_path = ':'.join(neutron.extensions.__path__)
 
```

Comment:
It is required for the test case to load the bnp_switch extension . It will be called by the extension framework